pkg/commands/models/stash_entry.go

```diff
@@ -1,17 +1,49 @@
 package models
 
-import "fmt"
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/jesseduffield/lazygit/pkg/utils"
+)
 
 // StashEntry : A git stash entry
 type StashEntry struct {
 	Index int
+	Time  *time.Time
 	Name  string
 }
 
 func (s *StashEntry) FullRefName() string {
 	return s.RefName()
 }
 
+func (s *StashEntry) hasTime() bool {
+	return s.Time != nil
+}
+
+func (s *StashEntry) Age() (string, error) {
+	if !s.hasTime() {
+		return "", errors.New("no time associated to this stash entry")
+	}
+	return utils.UnixToTimeAgo(s.Time.Unix()), nil
+}
+
+func (s *StashEntry) Line() string {
+	line := s.Name
+	if !s.hasTime() {
+		return line
+	}
+
+	age, err := s.Age()
+	if err != nil {
+		return line
+	}
+
+	return fmt.Sprintf("%s %s", age, line)
```

Comment:
This is presentation stuff, it doesn't belong in the model. But as I said above, we don't need any accessors anyway.